test/system_test.rb

```diff
@@ -2,6 +2,8 @@
 
 class SystemTest < LiveTest
   def around
+    Faktory::Testing.disable!
```

Comment:
Tests should not need to disable the testing APIs.  The testing tests should have to explicitly enable the Testing API.  Whew.